src/app/model/schema.ts

```diff
@@ -30,7 +30,7 @@ export class OrganizationInterface {
   /* Firebase Id, created automatically*/
   uid!: string;
   /*Name of the organization - eg Feed Folks */
-  name!: string;
+  displayName!: string;
   /*Name of the organization chapter eg Feed Folks - Studio City
     (make this the same as 'name' if org doesn't have chapters) */
```

Comment:
this comment should be fixed I guess.. ps not sure why we decided displayName over name but I guess as long as we are consistent.. 

actually on further thought I think name was gonna be eg "Feed Folks Coop LLC" or something and then displayName just "FeedFolks"... but this is fine as long as it works and if u think it's more consistent

Reply:
yeh I like name better, but displayName is what we're using on all the other objects so just keeping consistent